flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/RuntimeMonitorHandler.java

```diff
@@ -113,7 +117,17 @@ private void respondAsLeader(ChannelHandlerContext ctx, Routed routed, ActorGate
 		DefaultFullHttpResponse response;
 
 		try {
-			String result = handler.handleRequest(routed.pathParams(), jobManager);
+			Map<String, String> params = routed.pathParams();
```

Comment:
I was not aware of this particular aspect of REST. How should query params be represented? Since here, the resource is `/jars/:jarid`, and representing a plan or submit as `/jars/:jarid/:parallelism/:entry-class/...` doesn't seem okay.